src/app/src/components/MeetTheFishButton.js

```diff
@@ -38,18 +87,15 @@ const MeetTheFishButton = ({ fish, index, disabled, onButtonClick }) => {
         <>
             <FishButton
                 key={fish.commonName}
+                className={isFishLarge && 'is-large'}
```

Comment:
Instead of growing the fish now, we make their label opaque and give them a black background. Lets rename these variables and related classes to describe that better -- maybe something generic like `isFishAnimated`, `setIsFishAnimated` with class `.is-animated`. Or `isFishSelected`..etc...🤷‍♀